Stripe/STPPaymentCardTextField.m

```diff
@@ -476,24 +633,155 @@ - (void)setText:(NSString *)text inField:(STPCardFieldType)field {
         case STPCardFieldTypeCVC:
             textField = self.cvcField;
             break;
+        case STPCardFieldTypePostalCode:
+            textField = self.postalCodeField;
+            break;
     }
     textField.text = nonNilText;
 }
 
+- (CGFloat)numberFieldFullWidth {
+    // Current longest possible pan is 16 digits which our standard sample fits
+    if ([self.viewModel validationStateForField:STPCardFieldTypeNumber] == STPCardValidationStateValid) {
+        return [self widthForCardNumber:self.viewModel.cardNumber];
+    }
+    else {
+        return MAX([self widthForCardNumber:self.viewModel.cardNumber],
+                   [self widthForCardNumber:self.viewModel.defaultPlaceholder]);
+    }
+}
+
+- (CGFloat)numberFieldCompressedWidth {
+    // Current longest possible longest pan fragment is 5 characters
+    return [self widthForText:@"88888"];
```

Comment:
Ah maybe. I'm not sure how the linter detects these honestly. It may only trigger on strings literals that it can tell are passed directly to UIKit properties? I'll mark them as nonlocalized for clarity.